### PR TITLE
Docs: Update Document Content Security Policy options

### LINKED ISSUE #30411: Docs: Document content_security_policy setting
We missed adding this setting to the configuration docs in the main PR.

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Update docs/sources/administration/configuration.md with Content Security Policy options.

Fixes #30411.

### CODE REVIEW

File: docs/sources/administration/configuration.md [hunk 1]
```diff
@@ -508,6 +508,14 @@ Set to `true` to enable the X-Content-Type-Options response header. The X-Conten
 
 Set to `false` to disable the X-XSS-Protection header, which tells browsers to stop pages from loading when they detect reflected cross-site scripting (XSS) attacks. The default value is `false` until the next minor release, `6.3`.
 
+### content_security_policy
+
+Set to `true` to add the Content-Security-Policy header to your requests. CSP allows to control resources the user agent is allowed to load and helps prevent XSS attacks.
```
Comment (achatterjee-grafana) [inline docs/sources/administration/configuration.md:513]:
```suggestion
Set to `true` to add the Content-Security-Policy header to your requests. CSP allows you to control resources that the user agent can load and helps prevent XSS attacks.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added a copy0edit suggestion.

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -508,6 +508,14 @@ Set to `true` to enable the X-Content-Type-Options response header. The X-Conten
 
 Set to `false` to disable the X-XSS-Protection header, which tells browsers to stop pages from loading when they detect reflected cross-site scripting (XSS) attacks. The default value is `false` until the next minor release, `6.3`.
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:509]:
```suggestion
Set to `false` to disable the X-XSS-Protection header, which tells browsers to stop pages from loading when they detect reflected cross-site scripting (XSS) attacks. The default value is `true`.
```

Comment (aknuds1) [reply docs/sources/administration/configuration.md:509]: @oddlittlebird Please fix in another PR, it's unrelated to this one.

**Review by oddlittlebird — APPROVED** [review]

**Comment by oddlittlebird** [comment]
@aknuds1, I tried restarting your build, but the test is still failing.

**Pull request merged**